src/indexer/index_writer.rs

```diff
@@ -662,6 +736,33 @@ mod tests {
     use Index;
     use Term;
 
+    #[test]
+    fn test_operations_group() {
+        let mut schema_builder = schema::Schema::builder();
+        let text_field = schema_builder.add_text_field("text", schema::TEXT);
+        let index = Index::create_in_ram(schema_builder.build());
+        let mut index_writer = index.writer(3_000_000).unwrap();
```

Comment:
by default, tantivy uses several indexing threads. Can you force it to 1 in these unit tests?... Or 2?

As is, the unit test will fail on CPUs with a lot of cores, becasue the amount of allocated memory per thread will be lower than the lowest allowed amount.

Reply:
I do not know how to force it to use fewer threads. How would this be done?

Reply:
there is a method called something like ".write_with_num_threads(1, 3_000_000)" or something like that. You can check other unit tests. Most of them should be calling it.